buildtest/cli/commands.py

```diff
@@ -0,0 +1,29 @@
+def commands_cmd():
```

Comment:
can you rename this method and all instances where this is called to `list_buildtest_commands`